pkg/gardenlet/controller/backupentry/migration/reconciler.go

```diff
@@ -98,9 +58,9 @@ func (r *migrationReconciler) Reconcile(ctx context.Context, req reconcile.Reque
 	}
 
 	// If the backup entry is being deleted or no longer being migrated to this seed, clear the migration start time
-	if backupEntry.DeletionTimestamp != nil || !controllerutils.BackupEntryIsBeingMigratedToSeed(ctx, r.gardenClient, backupEntry, confighelper.SeedNameFromSeedConfig(r.config.SeedConfig)) {
+	if backupEntry.DeletionTimestamp != nil || !controllerutils.BackupEntryIsBeingMigratedToSeed(ctx, r.GardenClient, backupEntry, confighelper.SeedNameFromSeedConfig(r.Config.SeedConfig)) {
```

Comment:
Can you delete this `controllerutils.BackupEntryIsBeingMigratedToSeed` function and rather re-use the logic that is also part of this `isBeingMigratedPredicate` predicate?